src/background.js

```diff
@@ -156,6 +275,7 @@ window.scrNoti.isFolderToCheck = async (folder) => {
 window.scrNoti.tryNbrTimes = async (fnct, nbrTime) => {
   async function tryNbrTimesInner(pos) {
     try {
+// BUG: Sometimes I get "TypeError: fnct is not a function"
```

Comment:
I don't see how this error is possible. Please find a fix or remove the comment. Create an issue if you want.

Reply:
Neither do I. I just found it once in the error console, so I thought, I'll mention it …

Reply:
Weird. Then please remove the comment and we'll create an issue if the error coms back!

Reply:
Done.